spec/presenters/spree/retail/shopify/part_presenter_spec.rb

```diff
@@ -1,8 +1,9 @@
 require 'spec_helper'
 
-module Shopify
+module Spree::Retail::Shopify
```

Comment:
IMHO this is fine in specs. I know you asked this question about whether we keep this, the inline version or the tree version and honestly it's really whatever. If you want to enforce one way to write specs in the codebase that's chill but, I don't feel like it's 'gross' if we all write with our preferred style. Totally up to you though. 👍